taxonomies/food/ingredients.txt

```diff
@@ -13819,13 +13819,13 @@ it: prosciutto cotto
 nl: gekookte ham
 pt: fiambre cozinhado
 sv: kokt skinka
-
```

Comment:
I moved the properties for "cooked ham" to the en:cooked-ham and removed the properties from its children, as they will be inherited.